cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsRequestExecutor.java

```diff
@@ -203,7 +203,9 @@ private Object computeQueueLength(MetricsRequest metricsRequest) {
       if (queueName.isStream()) {
         streamNames.add(queueName.getSimpleName());
       } else if (queueName.isQueue()) {
-        String context = String.format("%s.f.%s.%s",
+        // TODO: After CDAP-1167 is fixed, namespace should be the first component of the queueName.
```

Comment:
I will have to update this PR, since there will be a merge conflict after merging the CDAP-1167 PR with develop.